css/components/badge.css

```diff
@@ -0,0 +1,82 @@
+.badge {
+    position: relative;
+    display: inline-flex;
+    flex-shrink: 0;
+    vertical-align: middle;
+}
+
+.badge .is-text {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+
+    position: absolute;
+    top: 0;
+    right: 0;
+    line-height: 1;
+    z-index: 1;
+    
+    border-radius: var(--radius-rounded);
+    font-size: var(--size-7);
+    height: var(--size-10);
+    min-width: var(--size-10);
+    padding: var(--size-0) var(--size-4);
+}
+
+/* ---- Colors ---- */
+
+.badge .is-primary {
+    background-color: var(--primary);
+    color: var(--white);
+}
+
+.badge .is-secondary {
+    background-color: var(--secondary);
+    color: var(--white);
+}
+
+/* ---- Position ---- */
+
+/* NOTE: No idea how the below code works */
```

Comment:
you can use css grid for achiveing that top , left , right positioning by the idea of overlapping

Reply:
Will consider doing that in future.